Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -65,58 +65,55 @@
     <constant name="FwdTracker_z2" value="2000*mm"/>
     <constant name="FwdTracker_zOffset" value="14000*mm"/>
```

Comment:
Hi! Thanks for the numbers. A couple of small detail questions:
- Why has rMax changed (out of curiosity)?
- Technically the forward tracker extrudes from the envelope given by Werner by the width of one forward disk
Can you make sure the corresponding correct numbers end up in Werner's "master document"?